drivers/regulator/regulator_da1469x.c

```diff
@@ -281,6 +282,16 @@ static int regulator_da1469x_set_voltage(const struct device *dev, int32_t min_u
 	uint16_t idx;
 	uint32_t mask;
 
+	if (dev == DEVICE_DT_GET(DT_NODELABEL(vdd)) && min_uv < 1200000) {
+		/*
+		 * Check if the current clock is PLL and abort as PLL requires
+		 * that VDD be 1.2V.
+		 */
+		if (SystemCoreClock == 96000000) {
+			return -EPERM;
+		}
+	}
```

Comment:
we only need to reject change if running on PLL and max_uv is <= 1.2V

if running on PLL and min_uv is < 1.2V it should be set to 1.2V to make sure we select proper voltage below

Reply:
Fixed

Reply:
min_uv needs to be adjusted as otherwise it will fail on set_voltage(0.9V, 1.2V), i.e. for 0.9V will be selected for PLL. get_win_index below seems to select min value from a range.

Reply:
@ioannis-karachalios the last comment (re `min_uv`) is not addressed

Reply:
Don't know why but I missed your comment on `min_uv`. It should be fixed now.